.ci/config/stage-types.yaml

```diff
@@ -35,25 +35,13 @@ engine-unit:
   - 'postgresql'
   - 'cockroachdb'
   - 'authorizations'
-engine-unit-authorizations:
-  directory: 'engine'
-  command: 'clean test -PcfgAuthorizationCheckRevokesAlways,'
+db-unit-authorizations:
+  directory: '.'
+  command: 'clean test -pl "engine,webapps" -Dskip.frontend.build=true -PcfgAuthorizationCheckRevokesAlways,'
   stash:
     runtimeStash: true
   labels:
   - 'authorizations'
```

Comment:
This means that now, the `webapp-unit-authorizations` tests will also run on PRs with the `authorizations` label. I guess we are okay with this?

Reply:
As discussed, this is okay.